distribution/docs/src/main/resources/_contents/_sources/registry-store-contents.adoc

```diff
@@ -0,0 +1,26 @@
+==== Registry Store
+
+[NOTE]
+====
+The Registry Store is currently marked *Experimental*. While functional and tested, it is subject to change or removal during the incubation period.
+====
+
+{description}
+
+===== Installing Registry Store
+
+The Registry Store is is not installed by default with a standard installation in the ${ddf-registry} application.
+
+===== Configuring Registry Store
+
+To configure the Registry store:
+
+. Navigate to the ${admin-console}.
+. Select *${ddf-registry}*.
+. Select *Registry Store*.
```

Comment:
Instead of Select Registry Store, you could have:
. Select the Remote Registries Tab and click the *Add* button.

-or-
. Select the Configuration Tab and select *Registry Store*.